packages/synthetix-main/contracts/modules/core/AssociatedSystemsModule.sol

```diff
@@ -0,0 +1,53 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-contracts/contracts/ownership/OwnableMixin.sol";
+import "@synthetixio/core-contracts/contracts/proxy/UUPSProxy.sol";
+import "../../interfaces/IAssociatedSystemsModule.sol";
+import "../../storage/AssociatedSystemsStorage.sol";
+
+import "@synthetixio/core-contracts/contracts/interfaces/IUUPSImplementation.sol";
+import "@synthetixio/core-modules/contracts/interfaces/IOwnerModule.sol";
+import "@synthetixio/core-modules/contracts/interfaces/ITokenModule.sol";
+
+contract AssociatedSystemsModule is IAssociatedSystemsModule, OwnableMixin, AssociatedSystemsStorage {
+    string public constant KIND_ERC20 = "erc20";
+    string public constant KIND_ERC721 = "erc721";
```

Comment:
Kind ERC721 (and KIND_OTHER) are unused. In fact in the code, Account that is an NFT is registered as ERC20